README.md

```diff
@@ -552,6 +567,13 @@ SPACESHIP_RUST_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"
 SPACESHIP_RUST_SYMBOL="𝗥 "
 SPACESHIP_RUST_COLOR="red"
 
+# JULIA
+SPACESHIP_JULIA_SHOW="${SPACESHIP_JULIA_SHOW:=true}"
```

Comment:
without `"${SPACESHIP_JULIA_*:=...}"`, please
check out other options for examples